CHANGES

```diff
@@ -7,7 +7,23 @@
  https://github.com/openssl/openssl/commits/ and pick the appropriate
  release branch.
 
- Changes between 1.1.1 and 1.1.2 [xx XXX xxxx]
+ Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
+
+  *) Change the possible version information given with OPENSSL_API_COMPAT.
+     It may be a pre-3.0.0 style numerical version number as was defined in
+     1.1.0, and it may also simply take the major version number.
+
+     Because of the practiced version numbering of pre-3.0.0 releases, the
```

Comment:
Because of the version numbering used by pre-3.0.0 releases, ...

Reply:
Okie